casts/bars/src/useNavigationHeaderLeftItems.tsx

```diff
@@ -15,7 +15,16 @@ function findIfCanGoBackForStack(state: NavigationState, parentState?: Navigatio
     }
 
     if (state.type === 'split') {
-        const { nestedStack } = state as { nestedStack?: number[] };
+        const { nestedStack, isSplitted, history } = state as any as {
+            nestedStack?: number[];
+            isSplitted: boolean;
+            history: number[];
+        };
```

Comment:
It's actually because I had to cast it due to inability to use typings from split navigator, as it introduces cycles. So this type isn't going to be re-used and is going to be used in this particular place only.